views/restaurant-home.html

```diff
@@ -0,0 +1,429 @@
+                    <img src="https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_508,h_320,c_fill/lite97zgd44crsd6znxo"
+                        alt="chinese work" loading="lazy">
+                    <div class="card-badge">
+                        <span>Pro- Get 20% off</span>
+                    </div>
+                </div>
+                <div class="ui-card-list-content">
+                    <span class="font-bold ">Chinese Wok (Wok Express) </span>
+                    <div>
+                        <span>&#x20b9;</span>
+                        <span class="font-md-light">254</span>
+                    </div>
+                    <span class="font-md-light">Light eater - choice of any 1 rice/noodles, 1 sauce, 4
+                        vegetables and 2 seasoning.</span>
+                </div>
+                <div class="ui-card-list-button flex flex-column gap">
+                    <button
+                        class="btn-outline-primary btn btn-py-1 flex  ui-card-small-override-prop">
+                        <span class="icon-default-pl">Add To Cart</span>
+                        <span class="material-icons icon-sm">add</span>
+                    </button>
+                    <button
+                    class="btn-outline-primary btn btn-py-1 flex gap-1 ui-card-small-override-prop">
+                    <span class="icon-default-pl">Add To wishlist</span>
+                   
+                </button>
+                </div>
+            </div>
+
+            <div class="ui-card-list card-py gap shadow-sm mb-1">
+                <div class="ui-card-list-img relative">
+                    <img src="https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_508,h_320,c_fill/lite97zgd44crsd6znxo"
+                        alt="chinese work" loading="lazy">
+                    <div class="card-badge">
+                        <span>Pro- Get 20% off</span>
+                    </div>
+                </div>
+                <div class="ui-card-list-content">
+                    <span class="font-bold ">Chinese Wok (Wok Express) </span>
+                    <div>
+                        <span>&#x20b9;</span>
+                        <span class="font-md-light">254</span>
+                    </div>
+                    <span class="font-md-light">Light eater - choice of any 1 rice/noodles, 1 sauce, 4
+                        vegetables and 2 seasoning.</span>
+                </div>
+                <div class="ui-card-list-button flex flex-column gap">
+                    <button
+                        class="btn-outline-primary btn btn-py-1 flex  ui-card-small-override-prop">
+                        <span class="icon-default-pl">Add To Cart</span>
+                        <span class="material-icons icon-sm">add</span>
+                    </button>
+                    <button
+                    class="btn-outline-primary btn btn-py-1 flex gap-1 ui-card-small-override-prop">
```

Comment:
what does this class do: "btn-py-1"?

suggestion: class names can be more descriptive